proto/cosmos/protocolpool/v1/tx.proto

```diff
@@ -29,6 +29,15 @@ service Msg {
 
   // ClaimBudget defines a method to claim the distributed budget.
   rpc ClaimBudget(MsgClaimBudget) returns (MsgClaimBudgetResponse);
+
+  // AddContinuousFunds defines a method to add funds continuously.
+  rpc AddContinuousFunds(MsgAddContinuousFunds) returns (MsgAddContinuousFundsResponse);
+
+  // FundDispensationProposal defines a method for funds dispensation.
+  rpc FundDispensationProposal(MsgFundDispensationProposal) returns (MsgFundDispensationProposalResponse);
```

Comment:
Can we get the naming consistent?

`FundDispensationProposal`  / `CancelContinuousFundProposal` isn't consistent.

Reply:
we could do create fundproposal as a name to keep it simple?

Reply:
I think they are just duplicate